Xlib/protocol/display.py

```diff
@@ -42,12 +42,41 @@
 from . import event
 
 if PY3:
-    def buffer(object, offset=None, size=None):
+
+    class bytesview(object):
+
+        def __init__(self, data, offset=None, size=None):
+            if offset is None:
+                offset = 0
+            if size is None:
+                size = len(data)-offset
+            if isinstance(data, bytes):
+                view = memoryview(data)
+            elif isinstance(data, bytesview):
+                view = data._view
```

Comment:
All static analyzers will produce a warning for this line. Accessing pseudo-private member of another object can be eliminated with a read-only property `view` for the `_view` field (as an option). Or you may make it public for simplicity.